Modules/Microsoft365DSC/DSCResources/MSFT_AADConditionalAccessPolicy/MSFT_AADConditionalAccessPolicy.psm1

```diff
@@ -1125,55 +1125,49 @@ function Set-TargetResource
             }
         }
         Write-Verbose -Message 'Set-Targetresource: process includeGuestsOrExternalUsers'
-        $includeGuestsOrExternalUsers = @{}
-        [string]$IncludeGuestOrExternalUserTypes = $IncludeGuestOrExternalUserTypes -join ','
-        $includeGuestsOrExternalUsers.Add('guestOrExternalUserTypes', $IncludeGuestOrExternalUserTypes)
-        $externalTenants = @{}
-        if ($IncludeExternalTenantsMembershipKind -eq 'All')
-        {
-            $externalTenants.Add('@odata.type', '#microsoft.graph.conditionalAccessAllExternalTenants')
-        }
-        elseif ($IncludeExternalTenantsMembershipKind -eq 'enumerated')
-        {
-            $externalTenants.Add('@odata.type', '#microsoft.graph.conditionalAccessEnumeratedExternalTenants')
-        }
-        else
-        {
-            $externalTenants.Add('@odata.type', '')
-        }
-        $externalTenants.Add('membershipKind', $IncludeExternalTenantsMembershipKind)
-        if ($IncludeExternalTenantsMembers)
-        {
-            $externalTenants.Add('members', $IncludeExternalTenantsMembers)
+        if($IncludeGuestOrExternalUserTypes){
```

Comment:
Please follow the code styline guidelines for this project by placing a space behind the if-statement and the open brace on the next line. Also, wouldn't it be better to check if the count is not equal to zero:

```PowerShell
if ($IncludeGuestOrExternalUserTypes.Count -ne 0)
{
    # The code here
}
```